### ISSUE TITLE
Feature Request: "Session" 's session management

### ISSUE THREAD

**Issue by RagueI** [oxen-io/session-desktop#2180]
it would be fantastic if users can manage the logged in session on each device.

For instance, user may decide to revoke a particular session on a particular device to ensure security.

**Comment by BirdInFire** [comment]
> it would be fantastic if users can manage the logged in session on each device.
> 
> For instance, user may decide to revoke a particular session on a particular device to ensure security.

Yeah and an enrollment like on signal (by Qr-code) can be also very appreciated.
Privacy without security mean nothing.

**Comment by KeeJef** [comment]
I think the first part of this would be to show which devices are sharing a particular Session ID (Linked), removing accounts would be tricky, but providing visibility is the first step
